### PR TITLE
Fix/update ikfast plugin

### LINKED ISSUE #83: Re-generate ikfast plugin
### Commit
3efae68

### Steps to reproduce
How can the issue/bug be reproduced?
1. `roslaunch prbt_moveit_config moveit_planning_execution.launch`

### Expected behavior
No warnings appear.

### Observed behavior
Log output:
```
[ WARN] [1552987873.840199096]: Kinematics solver doesn't support #attempts anymore, but only a timeout.
Please remove the parameter '/rviz_ROS_PC_6811_9210959308488888664/manipulator/kinematics_solver_attempts' from your configuration.
[ WARN] [1552987873.848169752]: IK plugin for group 'manipulator' relies on deprecated API. Please implement initialize(RobotModel, ...).
```

### Reference to test
Maybe a test checking for warning in /rosout during normal launch would help?

The fix should be as easy as re-generating the plugin from the current moveit master (see [moveit commit f19e411](https://github.com/ros-planning/moveit/commit/f19e411f2c4464783fd3994e6d3f6600adca7c91#diff-c0919d8a62d0d141b12acc2bf76ba7ed)

Related to #44  (hotfix)

### PR DESCRIPTION
**Pull request by ct2034**
Fixes #83

### CODE REVIEW

File: prbt_ikfast_manipulator_plugin/package.xml
Comment (martiniil) [inline prbt_ikfast_manipulator_plugin/package.xml]: I saw a commit removing this dependency. Which moveit version did you have when you ran the update? current is 1.0.1

Comment (ct2034) [reply prbt_ikfast_manipulator_plugin/package.xml]: This is necessary for the RobotModelLoader

Comment (martiniil) [reply prbt_ikfast_manipulator_plugin/package.xml]: Maybe this is a good point for migrating from package format 1.0 -> 2.0? Then it would suffice to declare `moveit_ros_planning` as test_depend

Comment (ct2034) [reply prbt_ikfast_manipulator_plugin/package.xml]: not according to catkin_lint. You still need it as build_depend. I switched to 2.0 now, because this is what the moveit template / script expects

File: prbt_ikfast_manipulator_plugin/CMakeLists.txt
Comment (jschleicher) [inline prbt_ikfast_manipulator_plugin/CMakeLists.txt]: Please move the test_depends into the `if(CATKIN_ENABLE_TESTING)` section.
Then `catkin_lint` should allow dropping it as `<build_depend>`

File: prbt_ikfast_manipulator_plugin/package.xml
Comment (jschleicher) [inline prbt_ikfast_manipulator_plugin/package.xml:21]: could be combined into a single `<depend>`, see http://docs.ros.org/jade/api/catkin/html/howto/format2/migrating_from_format_1.html

**Review by jschleicher — CHANGES_REQUESTED** [review]

File: prbt_ikfast_manipulator_plugin/package.xml
Comment (ct2034) [reply prbt_ikfast_manipulator_plugin/package.xml:21]: yes, but the ikfast script does not recognize this and would add the build_depends anyway
I was trying to make it easier to update using the script and require less manual changes after the script is run.

File: prbt_ikfast_manipulator_plugin/CMakeLists.txt
Comment (ct2034) [reply prbt_ikfast_manipulator_plugin/CMakeLists.txt]: good point!

**Review by jschleicher — APPROVED** [review]

**Review by martiniil — CHANGES_REQUESTED** [review]
The travis log still shows some catkin_lint errors. And the changelog has to be updated.

**Comment by ct2034** [comment]
> The travis log still shows some catkin_lint errors. And the changelog has to be updated.

Which changelog?

**Comment by martiniil** [comment]
the changelog in prbt_ikfast_manipulator_plugin

**Review by martiniil — APPROVED** [review]

**Comment by martiniil** [comment]
Please wait until kinetic is released and merged into melodic before you merge this.

**Comment by martiniil** [comment]
@ct2034 Please rebase the branch on melodic-devel

**Pull request merged**
